File: lat1_c++/soccer.cpp
#include <iostream>
#include <string>

using namespace std;

class soccer{
private:
	string club;
	string country;
	string year;
	string player;
	string stadium;

public:
	soccer(){
		this->club = "";
		this->country = "";
		this->year = "";
		this->player = "";
		this->stadium = "";
	}

	soccer(string club, string country, string year, string player, string stadium){
		this->club = club;
		this->country = country;
		this->year = year;
		this->player = player;
		this->stadium = stadium;
	}

	void setClub(string club){
		this->club = club;
	}

	void setCountry(string country){
		this->country = country;
	}

	void setYear(string year){
		this->year = year;
	}

	void setPlayer(string player){
		this->player = player;
	}

	void setStadium(string stadium){
		this->stadium = stadium;
	}

	string getClub(){
		return club;
	}

	string getCountry(){
		return country;
	}

	string getYear(){
		return year;
	}

	string getPlayer(){
		return player;
	}

	string getStadium(){
		return stadium;
	}

	~soccer(){}
};

File: lat1_java/Soccer.java
public class Soccer {
	private String club;
	private String country;
	private String year;
	private String player;
	private String stadium;

	public Soccer(String club, String country, String year, String player, String stadium){
		this.club = club;
		this.country = country;
		this.year = year;
		this.player = player;
		this.stadium = stadium;
	}

	//setter
	public void setClub(String club){
		this.club = club;
	}

	public void setCountry(String country){
		this.country = country;
	}

	public void setYear(String year){
		this.year = year; 
	}

	public void setPlayer(String player){
		this.player = player;
	}

	public void setStadium(String stadium){
		this.stadium = stadium;
	}

	//getter
	public String getClub(){
		return club;
	}

	public String getCountry(){
		return country;
	}

	public String getYear(){
		return year;
	}

	public String getPlayer(){
		return player;
	}

	public String getStadium(){
		return stadium;
	}	

}